arkouda/io.py

```diff
@@ -1229,194 +1215,30 @@ def read(
     Read with file Extension
     >>> x = ak.read('path/name_prefix.h5') # load HDF5 - processing determines file type not extension
     Read without file Extension
```

Comment:
not super important, but maybe we should remove the line `Read without file extension`, since that's no longer supported

Reply:
I'm gonna go ahead and merge since the actual code in the example has been updated, I don't think this one line comment is worth holding this up